README.md

```diff
@@ -8,7 +8,22 @@
 
 A DIDComm mediator is a cloud agent that facilitates secure, decentralized communication between mobile agents in the self-sovereign identity (SSI) ecosystem. This mediator acts as a cloud-based intermediary, enabling mobile agents to communicate without reliance on centralized infrastructures like Facebook, Signal, or Telegram.
 
-For further understanding checkout the [docs](https://github.com/adorsys/didcomm-mediator-rs/blob/222-readme.md-refactoring/docs/mediator-doc.md)
+For further understanding checkout the [docs](https://github.com/adorsys/didcomm-mediator-rs/blob/222-readme.md-refactoring/docs/mediator-doc.md)<br>
+
+###  Project Feature Implementation Tracker
+
+|Message types  | Spec Status | Implementation Status|
```

Comment:
Can you make the ticks under the Implementation status green for the ones adopted(☑) and red for that not(☒)?

Reply:
okay